examples/spin-timer/Cargo.lock

```diff
@@ -6031,6 +6046,24 @@ dependencies = [
  "wit-parser",
 ]
 
+[[package]]
+name = "wit-parser"
+version = "0.209.1"
```

Comment:
I don't quite understand why this `wit-parser` change is in the Cargo.lock file. If anyone has any insights, would appreciate it.

Reply:
I think this is just "we no longer enforce updating example lockfiles in CI so they tend to happen when the spirit moves rust-analyzer."